packages/dev/docs/pages/react-spectrum/ssr.mdx

```diff
@@ -52,30 +52,63 @@ Finally, when using server side rendering, the `locale` prop should be set expli
 
 [Next.js](https://nextjs.org) is a framework for building websites and web applications with React. It supports both server side rendering as well as static rendering. In addition to using an `SSRProvider`, a small amount of configuration is required to get React Spectrum’s CSS working with Next.js.
 
-First, you’ll need to install a couple Next.js plugins:
+First, you’ll need to install an additional Next.js plugin:
 
 ```
-yarn add next-compose-plugins @zeit/next-css next-transpile-modules
+yarn add next-transpile-modules
 ```
 
-With these installed, add the following to your `next.config.js` file. This will ensure that React Spectrum’s CSS is loaded properly by Next.js.
+With this installed, add the following to your `next.config.js` file. This will ensure that React Spectrum’s CSS is loaded properly by Next.js. Note that packages may need to be removed or added to the config below if using an older or newer version of React Spectrum.
 
 ```tsx
-const withPlugins = require('next-compose-plugins');
-const withCSS = require('@zeit/next-css');
-const withTM = require('next-transpile-modules')([
-  '@adobe/react-spectrum',
-  '@spectrum-icons/.*',
-  '@react-spectrum/.*'
+const withTM = require("next-transpile-modules")([
```

Comment:
Just for my own knowledge: the star patterns don't work any more here?

Reply:
Correct; it seems like it is not possible since v5 of this plugin ([Source](https://github.com/martpie/next-transpile-modules/issues/149#issuecomment-746290761)). I believe these docs were written at v4. There are some alternate solutions in the thread, so if people feel strongly about implementing one of those, that could be an option.

Reply:
You might want to explain what is going on here. Is this just a list of everything you use in your project? Would you really have the monopackage and a bunch of the individual ones too?

Reply:
Good idea, I'll add that. I wasn't able to get individual packages working. It looks like it requires every one to be included, as well as the monopackage.

Reply:
Leaving as is for now, as to not cause extra confusion. We can follow up if we manage to get individual packages working.